src/libs/Navigation/AppNavigator/Navigators/CentralPaneNavigator/BaseCentralPaneNavigator.tsx

```diff
@@ -44,7 +45,7 @@ function BaseCentralPaneNavigator() {
             />
             <Stack.Screen
                 name={SCREENS.SEARCH.CENTRAL_PANE}
-                // We do it this way to avoid adding the url params to url
+                initialParams={{sortBy: CONST.SEARCH_TABLE_COLUMNS.DATE, sortOrder: 'asc'}}
```

Comment:
Let's use a const for `asc`, `desc`

Reply:
Also the default here should be `desc`

Reply:
Should I make the change where it is always `desc` first when you press on a header and `asc` on second press?

Reply:
Yes please